### PR TITLE
feat: allow backend to rename field of custom object

### LINKED ISSUE #4080: Settings/Objects/Field Edit - Allow custom field label edition

### PR DESCRIPTION
**Pull request by magrinj**
Closes #4080

### CODE REVIEW

File: packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts [hunk 1]
```diff
@@ -228,9 +228,23 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
           )
         : fieldMetadataInput;
 
-    const updatedFieldMetadata = await super.updateOne(id, updatableFieldInput);
+    const updatedFieldMetadata = await super.updateOne(id, {
```
Comment (charlesBochet) [inline packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts:231]: it looks to me that this allows the update of standard fields too:
In buildUpdatableStandardFieldInput, the name, desc, icon are not part of fieldMetadataInputOverrided

**Review by charlesBochet — COMMENTED** [review]
The code looks good except for the comment I have left!
I have tested and reviewed fieldMetadata + workspaceMigration + postgres schema, it looks good!

File: packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts (hunk 1 above)
Comment (magrinj) [reply packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts:231]: Hummm I think you're right, I've fixed it, sorry for that !

Comment (charlesBochet) [reply packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts:231]: I think your fix still allows to update the name of a standard field

**Comment by charlesBochet** [comment]
> The code looks good except for the comment I have left! I have tested and reviewed fieldMetadata + workspaceMigration + postgres schema, it looks good!

Actually, I've tested a more deeply and I was incorrect, good to go :)

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**